packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -252,8 +250,12 @@ function useShouldUpdateFocusOnMouseMove({ delayUpdateFocusOnHover, hidden }: IC
   return [shouldUpdateFocusOnMouseEvent, gotMouseMove, onMenuFocusCapture] as const;
 }
 
-function usePreviousActiveElement({ hidden, onRestoreFocus }: IContextualMenuProps, targetWindow: Window | undefined) {
-  const previousElementsQueue = React.useRef<HTMLElement[]>([]);
+function usePreviousActiveElement(
+  { hidden, onRestoreFocus }: IContextualMenuProps,
+  targetWindow: Window | undefined,
+  hostElement: any,
```

Comment:
Need access to the host element to know if the newly focused element is within that host (like focus moving to a searchbox)